clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -144,8 +144,9 @@ public interface ConsumerRebalanceListener {
     void onPartitionsRevoked(Collection<TopicPartition> partitions);
 
     /**
-     * A callback method the user can implement to provide handling of customized offsets on completion of a successful
-     * partition re-assignment. This method will be called after the partition re-assignment completes and before the
+     * A callback method the user is providing handling of cleanup up resource which are being assigned to other customers and
```

Comment:
This one isn't right. It's not usually used for cleanup and "customers" is not right. I suggest "A callback method the user can implement to be notified when partitions are assigned to this consumer. This method will be called after the partition re-assignment completes and before....."

Reply:
Thanks for review. Update already.